src/base/projects/Commit/History.svelte

```diff
@@ -31,7 +31,7 @@
   .commit {
     padding: 0.25rem 0;
   }
-  @media (max-width: 720px) {
+  @media (max-width: 960px) {
```

Comment:
This is a minor style improvement, the commit listing was changing its padding much later than the header and other components.